README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/marshallmcmullen/ebash.svg?branch=master)](https://travis-ci.org/marshallmcmullen/ebash)
```

Comment:
This is nifty b/c it will show the build status at the top of the README in GitHub repo.